test/e2e/apimachinery/garbage_collector.go

```diff
@@ -719,4 +725,105 @@ var _ = SIGDescribe("Garbage collector", func() {
 			framework.Failf("failed to wait for all pods to be deleted: %v", err)
 		}
 	})
+
+	// TODO: should be an integration test
+	It("[Feature:GarbageCollector] should support cascading deletion of custom resources", func() {
```

Comment:
Remove `[Feature:GarbageCollector]`, otherwise it's not run in the presubmit. See https://github.com/kubernetes/test-infra/blob/753266cb7d77456c2395521bece25eca51bfedcc/jobs/config.json#L9207

Looks like we have a few e2e tests that have never been running...I'll send a PR
```bash
$ git grep "\[Feature:GarbageCollector\]"
test/e2e/apimachinery/garbage_collector.go:495: It("[Feature:GarbageCollector] should keep the rc around until all its pods are deleted if the deleteOptions says so", func() {
test/e2e/apimachinery/garbage_collector.go:573: It("[Feature:GarbageCollector] should not delete dependents that have both valid owner and owner that's waiting for dependents to be deleted", func() {
test/e2e/apimachinery/garbage_collector.go:681: It("[Feature:GarbageCollector] should not be blocked by dependency circle", func() {
```